include/ibamr/IBFEMethod.h

```diff
@@ -794,9 +805,19 @@ class IBFEMethod : public IBStrategy
     /*
      * Data related to handling stress normalization.
      */
-    double d_epsilon;
+    double d_phi_epsilon;
+    double d_ipdg_jump0_penalty;
+    double d_ipdg_jump1_penalty;
+    double d_ipdg_beta0;
+    double d_ipdg_beta1;
+    libMesh::Order d_phi_fe_order;
+    double d_cg_penalty;
+    double d_phi_diffusion;
+    std::string d_phi_solver;
+    bool d_phi_current_config;
     bool d_has_stress_normalization_parts;
     std::vector<bool> d_is_stress_normalization_part;
+    double d_dt_previous;
```

Comment:
This is already provided by the `HierarchyIntegrator` base class via `HierarchyIntegrator::d_dt_previous`